pkgs/development/tools/rtags/default.nix

```diff
@@ -16,11 +16,13 @@ stdenv.mkDerivation rec {
            MACOSX_DEPLOYMENT_TARGET="10.9"
   '';
 
+  NIX_CFLAGS_COMPILE = "-fexceptions";
```

Comment:
Thx for the input, i am occasionally contributing to packages, so i am interrested to learn. You mean i should put FLAGS in CFLAGS or CXXFLAGS and not NIX_CFLAGS_COMPILE, right?